src/js/firstparties/twitter.js

```diff
@@ -63,13 +76,42 @@ function unwrapTwitterURLsInTimeline() {
   });
 
   // iterate another time to actually replace the link (of Twitter cards, e.g.)
-  findInAllFrames("a[href^='https://t.co/'], a[href^='http://t.co/'").forEach((link) => {
+  findInAllFrames(tcos_all).forEach((link) => {
     if (fixes.hasOwnProperty(link.href)) {
       unwrapTco(link, fixes[link.href]);
     }
   });
 }
 
+function unwrapTwitterPwaURLsInTimeline() {
+  // unwrap profile links
+  document.querySelectorAll(tcos_all).forEach((link) => {
+    // use cached value if possible
+    if (fixes.hasOwnProperty(link.href)) {
+      unwrapTco(link, fixes[link.href]);
+      return;
+    }
+
+    // find span element with link text in it
+    const allSpans = link.getElementsByTagName("span");
+    if (!allSpans) {
```

Comment:
I think we need to test `allSpans.length` here instead. An empty `HTMLCollection` is truthy.